### PR TITLE
Fixed Lunatone LM's Sol Shade

### PR DESCRIPTION
**Pull request by Grayahm**

### CODE REVIEW

**Comment by Grayahm** [comment]
How does this look?

`getterA (IS_ABILITY_BLOCKED) { Holder h->
              if (my.all.find{it.name == 'Solrock'} && h.effect.target.types.contains(R) && !h.effect.target.topPokemonCard.cardTypes.is(EX) && h.effect.ability instanceof PokePower) {
                h.object=true
              }
            }`

**Comment by ufodynasty** [comment]
It looks better, however on solrock, `}
          delayedA{
            after PLAY_CARD, {
              new CheckAbilities().run(bg)
            }
          }`
was used as the ability didn't immediately activate when any lunatone was put into play. For future reference, you can commit to the branch you  were working on in order to update a pr.

File: src/tcgwars/logic/impl/gen3/LegendMaker.groovy [hunk 1]
```diff
@@ -861,14 +861,12 @@ public enum LegendMaker implements LogicCardInfo {
         weakness P
         pokeBody "Sol Shade", {
           text "As long as you have Solrock in play, each player's [R] Pokémon (excluding Pokémon-ex) can't use any Poké-Powers."
-          delayedA {
             getterA (IS_ABILITY_BLOCKED) { Holder h->
```
Comment (ufodynasty) [inline src/tcgwars/logic/impl/gen3/LegendMaker.groovy:864]: Also perhaps fix your indentation?

File: src/tcgwars/logic/impl/gen3/LegendMaker.groovy [hunk 2]
```diff
@@ -861,14 +861,12 @@ public enum LegendMaker implements LogicCardInfo {
...
-              if (my.all.find{it.name == 'Solrock'} && h.effect.target.types.contains(R) && !h.effect.target.EX && h.effect.ability instanceof PokePower) {
+              if (my.all.find{it.name == 'Solrock'} && h.effect.target.types.contains(R) && !h.effect.target.topPokemonCard.cardTypes.is(EX) && h.effect.ability instanceof PokePower) {
```
Comment (ufodynasty) [inline src/tcgwars/logic/impl/gen3/LegendMaker.groovy:865]: .EX has the same functionality as topPokemonCard.cardTypes.is(EX). I prefer the the former as it keeps line length down but either is acceptable

**Comment by Grayahm** [comment]
Okay I think I fixed all of your suggestions, let me know though.

**Pull request merged**